File: official/docs/python/customs_items/create_customs_items.py
import easypost


easypost.api_key = "EASYPOST_API_KEY"

customs_item = easypost.CustomsItem.create(
    description="T-shirt",
    quantity=1,
    value=10,
    weight=5,
    hs_tariff_number="123456",
    origin_country="us",
)

print(customs_item)

File: official/docs/ruby/customs_items/create_customs_items.rb
require 'easypost'

EasyPost.api_key = ENV['EASYPOST_API_KEY']

customs_item = EasyPost::CustomsItem.create(
  {
    description: 'T-shirt',
    quantity: 1,
    weight: 5,
    value: 10,
    hs_tariff_number: '123456',
    origin_country: 'us',
  },
)

puts customs_item
